### ISSUE TITLE
hasRegisteredSite vs hasPrimarySite

### ISSUE THREAD

**Issue by VladimirAlexiev** [SEMICeu/Core-Business-Vocabulary#2]
I assume that one would capture "legal address" using org:hasRegisteredSite and "headquarters" using org:hasPrimarySite.

In https://github.com/SEMICeu/Core-Business-Vocabulary/issues/1 @philarcher correctly points out: 
>the registered address is often not the head office, typically it's the address of their lawyer or founder's home address or whatever. It's a legal address that does not have to bear any relation to the trading premises.

But the ORG ontology has this:
```ttl
org:hasRegisteredSite     rdfs:subPropertyOf org:hasPrimarySite;
```
which means "every registered site is a headquarter site".

If my assumption (that org:hasPrimarySite should be used to model "headquarters") is incorrect, then what should be used?

IMHO that rdfs:subPropertyOf  should be removed. If an address is both legal and headquarters, just use two parallel relations.

**Comment by makxdekkers** [comment]
I agree that the sub-property assertion is unhelpful. However, I am not aware of a mechanism to get W3C to consider changes to the Recommendation. 

I don't see the problem to use org:hasPrimarySite for the headquarters and at the same time use hasRegisteredSite. If the company is registered on a different address than the headquarters, you would have two primary sites. The spec states that hasPrimarySite "_Indicates **a** primary site for the Organization_" so it is not excluded that an organisation could have two primary sites (even if that sounds counterintuitive).
